spec/lib/common/models/collection_spec.rb

```diff
@@ -15,7 +16,7 @@
 
   it 'returns a JSON string whose keys' do
     json = JSON.parse(subject.to_json)
-    expect(json.first.keys).to contain_exactly(*%w(id first_name last_name birthdate zipcode))
+    expect(json.first.keys).to contain_exactly('id', 'first_name', 'last_name', 'birthdate', 'zipcode')
```

Comment:
boo 😞

Reply:
I could go either way on this one tbh. The splat is maybe a bit much though. The recommendation is more readable.